doc/cpd/101.rst

```diff
@@ -75,6 +78,16 @@ if set to ``True`` would use the secrets handler.
 The secrets handler would be automatically added to the ``StorageHandlerManager`` and
 would require no special configuration by the operator.
 
+New HTTP Handler
+~~~~~~~~~~~~~~~~
+A new handler called ``secrets`` could be added. This would proxy requests back to the
+Custodia instance.
```

Comment:
True.  I think I understand now how this all fits together, conceptually at least.